testgrid/config.yaml

```diff
@@ -2912,6 +2912,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-kubeadm-kinder-1-14-on-1-13
   - name: kubeadm-kinder-upgrade-1.13-1.14
     test_group_name: ci-kubernetes-e2e-kubeadm-kinder-upgrade-1-13-1-14
+  - name: kubeadm-kinder-external-etcd-1-14
+    test_group_name: ci-kubernetes-e2e-kubeadm-kinder-external-etcd-1-14
```

Comment:
i'm hesitating about maintaining these job as part of release dashboards before having the signal green for a while.
let's remove this for now.

we also have the `master` jobs under the `sig-release-master-informing` dashboard:
https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml#L2861-L2869

but we cannot add `kubeadm-kinder-external-etcd-master` there this late in the 1.15 cycle.

let's add the `kubeadm-kinder-external-etcd*` jobs to `sig-release-master-informing` and `sig-release-1.14-all` after the 1.15 release.

Reply:
Agreed, let's keep those job out of Sig release dashboard for now